css/style.css

```diff
@@ -0,0 +1,17 @@
+#img {
+    display: block;
+}
+
+#image {
+    width: 100%;
+    max-width: 100%;
+    height: auto;
+}
+
+.center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+    max-width: 100%;
+}
```

Comment:
Kindly follow best HTML and CSS practices for writing code. The indentation is not correct. Kindly implement Linters to check for styling errors before reviewing.